app/assets/stylesheets/_buttons.scss

```diff
@@ -37,7 +37,14 @@
       left: 10px;
     }
   }
-    
+
+  &.hb-octo-button {
+    img {
+      position: relative;
+      right: 7px;
+      bottom: 1px;
```

Comment:
Check out https://github.com/up-for-grabs/up-for-grabs.net/pull/163/files#diff-b8b01588d730dbc5770b98a7f4a45687R494 for how I styled a similar icon in a button without `position:relative`.